jetstream/external_config.py

```diff
@@ -37,32 +37,29 @@ class ExternalConfigCollection:
 
     configs: List[ExternalConfig] = attr.Factory(list)
 
-    JETSTREAM_CONFIG_REPO = "mozilla/jetstream-config"
+    JETSTREAM_CONFIG_URL = "https://github.com/mozilla/jetstream-config"
 
     @classmethod
     def from_github_repo(cls) -> "ExternalConfigCollection":
         """Pull in external config files."""
+        # download files to tmp directory
+        tmp_dir = Path(tempfile.mkdtemp())
+        repo = Repo.clone_from(cls.JETSTREAM_CONFIG_URL, tmp_dir)
```

Comment:
Using the jetstream-config tarball doesn't work here. The last_modified timestamps all get changed.

Reply:
👺 boooo, thanks for looking into it